mmv1/products/redis/api.yaml

```diff
@@ -143,6 +143,57 @@ objects:
           The ID of the instance or a fully qualified identifier for the instance.
         required: true
         input: true
+      - !ruby/object:Api::Type::NestedObject
+        name: persistenceConfig
+        description: Maintenance policy for an instance.
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: 'persistenceMode'
+            required: true
+            description: |
+              Optional. Controls whether Persistence features are enabled. If not provided, the existing value will be used.
+
+              - PERSISTENCE_MODE_UNSPECIFIED:	Not set.
```

Comment:
Can we remove the `PERSISTENCE_MODE_UNSPECIFIED` and `SNAPSHOT_PERIOD_UNSPECIFIED` values/references?

Reply:
done